### ISSUE TITLE
Part of Cerebellum missing in "desc-preproc_T1w" but not in "T1.mgz" (from freesurfer)

### ISSUE THREAD

**Issue by hjernemannen** [poldracklab/fmriprep#1528]
When T1.mgz (256x256x256) from the Freesurfer directory is converted to "*desc-preproc_T1w" (192x256x164), part of the cerebellum is removed in some of the subjects: 

![t1 mgz-desc-preproc_t1](https://user-images.githubusercontent.com/36411073/53954556-dad3bf80-40d6-11e9-849a-178e4faf8d26.PNG)

(Further, the cerebellum then gets distorted in the "*_space-MNI152NLin2009cAsym_desc-preproc_bold.nii.gz" image, an issue I believe is related part of Cerebellum missing in "*desc-preproc_T1w")

Any help would be highly appreciated.

**Comment by effigies** [comment]
Can you explain what you mean by converting `T1.mgz` to `desc-preproc_T1w.nii.gz`? What command are you running?

**Comment by hjernemannen** [comment]
Sorry about the confusion, the command I run is:
fmriprep $datadir/ ${destdir}/ participant -w ${workdir}/ --ignore {fieldmaps,slicetiming,sbref} ${distortionCorrection} ${templateResamplingGrid} --participant_label ${subId} --nthreads $nthreads

**Comment by effigies** [comment]
Okay. I'm a little confused, as we don't do any cropping of T1w images.

Is there a chance you could share your `T1w.nii.gz` from your source dataset, the `desc-preproc_T1w.nii.gz` from the fMRIPrep outputs, and your corresponding `T1.mgz` from the FreeSurfer directory? (All defaced, of course.)

**Comment by hjernemannen** [comment]
I found it strange as well, because I didn`t see any reason why you would want to crop the images. 

That sounds like a plan, thank you for being willing to look at them. Where do you want the files uploaded?

**Comment by effigies** [comment]
I don't think I have somewhere to anonymously upload, so whatever you feel comfortable with. [OnionShare](https://onionshare.org/), if you're feeling paranoid. :-)

**Comment by effigies** [comment]
I'm running this now to see if I can reproduce using fMRIPrep 1.3.1.

**Comment by hjernemannen** [comment]
Thank you very much

**Comment by effigies** [comment]
Running:

```
docker run --rm -it -e DOCKER_VERSION_8395080871=18.06.1-ce \
    -v /opt/freesurfer/license.txt:/opt/freesurfer/license.txt:ro \
    -v /data/bids/cerebellum_missing:/data:ro \
    -v /data/out/cerebellum_missing/derivatives:/out \
    -v /data/scratch/cerebellum_missing:/scratch \
    poldracklab/fmriprep:1.3.1 /data /out participant \
    --fs-no-reconall --notrack --output-space T1w --anat-only \
    --skip-bids-validation -w /scratch
```

I get a file with the same field-of-view as the original.
![preproc_t1w](https://user-images.githubusercontent.com/83442/53984527-7f5dfd80-40e7-11e9-94cd-03cf2c539c86.png)

I would suggest double-checking sure that your source dataset has a good T1w image, and then re-run with a fresh working directory (you can use `--anat-only --output-space T1w` to limit it to just running the basic T1w preprocessing) to see if you still produce this error.

Incidentally, what version of fMRIPrep are you using? We generally recommend people who've started preprocessing on one version stick with it unless errors make upgrading necessary, so if you have the time, it would be good to test with the version you've been using and make sure that it is replicable before upgrading.

**Comment by hjernemannen** [comment]
Again, thank you so much. I`ll try to run the analysis again using --anat-only --output-space T1w and my current version of fmriprep. I`m using fmriprep v1.2.5.

**Comment by hjernemannen** [comment]
I now found out that is was an error in one of my scripts, sorry about that. The script copied some old T1 files, that I was sure that I had deleted, with part of the cerebellum missing instead of the correct T1 files used in freesurfer. Thank you again for all the help and a quick reply, I highly appreciate it. Next time I`ll do a more thorough investigation of my own scripts and data used before asking.
